Dockerfile

```diff
@@ -0,0 +1,28 @@
+# Base image to use, this must be set as the first line
+FROM ubuntu
+
+LABEL maintainer="fyuxin@uwaterloo.ca"
+
+# Commands to install python
+RUN apt-get update \
+    && apt-get install -y python3-pip python3-dev vim curl apt-utils \
+    && cd /usr/local/bin \
+    && ln -s /usr/bin/python3 python \
+    && pip3 install --upgrade pip \
```

Comment:
Please unify `pip3` and `pip`. Choose one of them to use.

Reply:
Removed.